src/Core/ConfigurationDefaults.re

```diff
@@ -36,6 +36,12 @@ let getDefaultConfigString = configName =>
   "workbench.sideBar.location": "left",
   "workbench.sideBar.visible": true,
   "workbench.statusBar.visible": true,
+  "workbench.statusBar.items": {
+    "left": ["notificationCount", "macro", "leftItems", "diagnosticCount", "git"],
```

Comment:
It looks like we'll need to replace `leftItems` with `...` and `rightItems` with `...` below

Reply:
(Oops, this was just echoing what @z0al mentioned in his reply - beat me to it!)